### PR TITLE
fix - add ellipsis in sub-lists of MyCompany when content too long

### LINKED ISSUE #9490: Long usernames broken UI in subsections with users
Need to verify all subsection tables contains users (or approvers) in organization after fix.
image

### PR DESCRIPTION
**Pull request by Platonn**
The CSS ellipsis was working well for the main tables (i.e. the main users table), but was not working for the sub lists (i.e. list of Users for the Unit). When the sub-list's cell had too long content, text went beyond the borders.

Now it's fixed. The last column is sqhashed and aligned to the right. And the rest of preceding columns take proportionally he remaining space and are truncated with ellipsis, if their content is too long. 

Additional improvements:
- added `title` attributes to the cell components - for better desktop experience, to allow lookup of the full value
- reused templates in cell-components to avoid duplication
- dropped obsolete padding styling for single-cell link (it caused the unit address list to have inconsistent padding)
- added missing padding to the unit cell (before that, the ellipsis were squeezed too close to the unit cell)
- fix: aligned lines of ghost design (before they were at the very bottom of the sub-list cell)
- fix: marked 'unit' column as non-linkable - in unit's assigned approvers sub-list (before they were clickable and pointed to empty URL)

closes GH-9490

### CODE REVIEW

File: feature-libs/organization/administration/components/unit/list/toggle-link/toggle-link-cell.component.ts [hunk 1]
```diff
@@ -18,8 +18,6 @@ export class ToggleLinkCellComponent extends OrganizationCellComponent {
     return this.model.depthLevel;
   }
 
-  @HostBinding('class.content-wrapper') cls = true;
```
Comment (Platonn) [inline feature-libs/organization/administration/components/unit/list/toggle-link/toggle-link-cell.component.ts:21]: When I changed `.content-wrapper` to `display:inline`, it broke the rows of the unit tree. I fixed that, by removing this class from the toggle-link-cell component (used in unit tree).
I believe it's a right decision btw. becuase semantically this is not _content_ cell. And it looks well (same as before) without `.content-wrapper` class anyway.

**Review by Pucek9 — CHANGES_REQUESTED** [review]
Found 1 place where it doesn't seems ideal. I mean http://localhost:4200/powertools-spa/en/USD/organization/units/Rustic%20Services/approvers/assign

File: feature-libs/organization/administration/components/unit/list/toggle-link/toggle-link-cell.component.ts (hunk 1 above)
Comment (Pucek9) [reply feature-libs/organization/administration/components/unit/list/toggle-link/toggle-link-cell.component.ts:21]: I don't see any disadvantages of that change 👍

**Review by Pucek9 — CHANGES_REQUESTED** [review]
On mobile view in http://localhost:4200/powertools-spa/en/USD/organization/units/Rustic%20Services/users it seems like this:
image
Not sure we can close without fix that

**Review by Pucek9 — APPROVED** [review]
We decided to create new - stretch issue with that small visual defect in mobile #9561

**Pull request merged**